visualizer/flatten.js

```diff
@@ -69,11 +113,16 @@ Flatten.prototype.nodes = function () {
   // Flatten out the nodes, removed children of collapsed nodes and calculate
   // index.
   const nodes = [];
```

Comment:
The order of this might have changed. Possible fix is `stack.push(...node.children.reverse());`.